frontend/pages/index.tsx

```diff
@@ -159,43 +159,61 @@ class Splash extends React.Component<SplashProps, SplashState> {
   }
 
   resetDisplay(nameInput, selectedTags) {
-    const tagSelected = selectedTags.filter((tag) => tag.name === 'Tags')
-    const sizeSelected = selectedTags.filter((tag) => tag.name === 'Size')
-    const applicationSelected = selectedTags.filter(
-      (tag) => tag.name === 'Application',
-    )
-    let { clubs } = this.state
+    const tagSelected = selectedTags
+      .filter((tag) => tag.name === 'Tags')
+      .map((tag) => tag.value)
+    const sizeSelected = selectedTags
+      .filter((tag) => tag.name === 'Size')
+      .map((tag) => tag.value)
+
+    const requiredApplication =
+      selectedTags.findIndex(
+        (tag) => tag.name === 'Application' && tag.value === 1,
+      ) !== -1
+    const noRequiredApplication =
+      selectedTags.findIndex(
+        (tag) => tag.name === 'Application' && tag.value === 2,
+      ) !== -1
+    const acceptingMembers =
+      selectedTags.findIndex(
+        (tag) => tag.name === 'Application' && tag.value === 3,
+      ) !== -1
```

Comment:
Kinda messy but I couldn't come up with a better way to do this

Reply:
Hmm, I can't think of a better way to do this either without refactoring more broadly. The whole sidebar tag system could be refactored a bit to make it easier to extend, but that can be done in a later commit.

This works for me!